http/server.go

```diff
@@ -254,27 +258,27 @@ func getFedAddress(s *Server, netParams chaincfg.Params) (btcutil.Address, error
 	return fedAddress, nil
 }
 
-func (s *Server) getSignatureFromHash(hash string, hashBytes []byte) (string, []byte, []byte, []byte, error) {
+func (s *Server) getSignatureFromHash(hash string, hashBytes []byte) (string, string, []byte, string, error) {
```

Comment:
I think I already commented this in a previous PR, but this function is doing too many things: it computes the signature and it parses addresses. I think that the function should be split in two since one thing has nothing to do with the other.

Reply:
Yes and we had agreed to do it in another task/PR. Those changes weren't performed as I got to work into this issue with the derivation value. I just changed it now so you don't need to comment it in further PRs